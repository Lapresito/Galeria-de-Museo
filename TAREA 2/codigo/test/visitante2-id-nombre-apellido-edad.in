crearVisitante 31 Valquiria Sanchez 12
imprimirVisitante
imprimirEdadVisitante
imprimirIdVisitante
liberarVisitante
Fin
