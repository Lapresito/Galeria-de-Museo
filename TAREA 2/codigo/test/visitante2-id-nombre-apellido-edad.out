Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creadox de forma exitosa.
2> Visitante Valquiria Sanchez
Id: 31
Edad: 12
3> La edad del visitante es: 12
4> La id del visitante es: 31
5> Visitante liberado con exito.
6> Fin.
